Kernel/Arch/x86_64/ISABus/I8042Controller.cpp

```diff
@@ -6,17 +6,37 @@
 
 #include <Kernel/Arch/Delay.h>
 #include <Kernel/Arch/x86_64/IO.h>
-#include <Kernel/Arch/x86_64/ISABus/HID/PS2KeyboardDevice.h>
-#include <Kernel/Arch/x86_64/ISABus/HID/PS2MouseDevice.h>
 #include <Kernel/Arch/x86_64/ISABus/HID/VMWareMouseDevice.h>
 #include <Kernel/Arch/x86_64/ISABus/I8042Controller.h>
+#include <Kernel/Bus/SerialIO/PS2/Device.h>
 #include <Kernel/Devices/HID/KeyboardDevice.h>
 #include <Kernel/Devices/HID/MouseDevice.h>
```

Comment:
Not really, because they are used later on:
```c++
auto keyboard_device = TRY(KeyboardDevice::try_to_initialize());
auto error_or_device = PS2KeyboardDevice::try_to_initialize(*this, I8042PS2PortIndex::FirstPort, *keyboard_device);
```
and -
```c++
m_second_ps2_port.device_type = PS2DeviceType::StandardMouse;
auto mouse_device = TRY(MouseDevice::try_to_initialize());
auto vmmouse_device_or_error = VMWareMouseDevice::try_to_initialize(*this, I8042PS2PortIndex::SecondPort, *mouse_device);
```

Reply:
In #18792, which is a follow-up PR, we change this, so I will keep this as a reminder.